.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug Report
+about: Bug reports about the Solidity Compiler Infrastructure.
+---
+
+## Prerequisites
+
+- First, many thanks for taking part in the community. We really appreciate that.
+- We realize there is a lot of data requested here. We ask only that you do your best to provide as much information as possible so we can better help you.
+- Support questions are better asked in one of the following locations:
+	- [Solidity chat](https://gitter.im/ethereum/solidity)
+	- [Stack Overflow](https://ethereum.stackexchange.com/)
+- Ensure the issue isn't already reported.
+- Should be reproducible with the latest ``solc`` version.
```

Comment:
Should mention that this isn't a hard requirement, in some cases it may make sense reporting older ones, within reason.

Reply:
in that case I'd drop that line entirely, as we'd test it anyways (no matter what), and the `Environment` section already covers the solc version.